### ISSUE TITLE
Tutorial

### ISSUE THREAD

**Issue by giron3s** [Codigami/face4j#21]
Hello Nischal Shetty

I now i need to implement some function wherewith the user can write the post on his wall and his friends wall, can comment the one post, get the friend list .... 

I was looking for  for some tutorial, which explain how to do those with your api, but i can not found! Do you have some tutorials or could you explain, how can i write the one pots on my wall or on my friends wall?

Thanks,
Best wishes

**Comment by nischal** [comment]
Have you implemented authentication? That is the first step. Do as written in this article - https://github.com/nischal/face4j/wiki/Facebook-oAuth-login-with-face4j

**Comment by giron3s** [comment]
Yes, I could login in facebook and get the access token. I need the tutorial about how can i write the one pots on my wall or on my friends wall?

**Comment by nischal** [comment]
Look at the #wallPost method in class 'facebook'.

On 23 October 2012 13:33, giron3s notifications@github.com wrote:

> Yes, I could login in facebook and get the access token. I need the
> tutorial about how can i write the one pots on my wall or on my friends
> wall?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/nischal/face4j/issues/21#issuecomment-9693574.

## 

-Nischal
Founder, JustUnfollow
twitter: NischalShetty http://twitter.com/nischalshetty
facebook: Nischal http://facebook.com/nischal

http://www.justunfollow.com
JustUnfollow http://www.justunfollow.com - Twitter follower management
made easy
